content/en/security/application_security/threats/threat-intelligence.md

```diff
@@ -89,6 +91,30 @@ On a new [references table][4] page:
    
    {{< img src="/security/application_security/threats/threat_intel/threat_intel_ref_table_enabled.png" alt="Enabled reference table" style="width:100%;" >}}
 
+#### Using cloud storage
+
+When the reference table is created from cloud storage, it is refreshed periodically. The entire table is *replaced*. Data is not merged.
+
+See the related reference table documentation for:
+- [Amazon S3][11]
+- [Azure storage][12]
+- [Google Cloud storage][13]
+
+#### Troubleshooting cloud imports
+
+If the reference tables are not refreshing, select the **View Change Events** link from the settings on the reference table detail page. 
+
+**View Change Events** opens a page in **Event Management** showing potential error events for the ingestion. You can also filter in **Event Management** using the reference table name.
+
+<div class="alert alert-info">In Datadog Event Management, it can look like the data is fetched from the cloud, but it can take a few more minutes to propagate those changes to Threat Intellegence.</div>
+
+Other useful cloud import details to remember:
+
+- The expected latency before updated enrichments are available when a source is uploaded or updated is 10 to 30 minutes.
+- How to know when the updates are applied: The changes are visible in the reference table or in the spans. Select the **View Change Events** link from settings on the reference table detail page to see the related events.
+- The update replaces the *entire table* with the new data. 
+- In the case of a duplicated primary key, the rows with the duplicated key are not written, and an error is shown in the reference table detail page.
```

Comment:
```suggestion
- In case of a duplicated primary key, the rows with the duplicated key are not written, and an error is shown in the reference table detail page.
```